tests/bearlib/abstractions/LinterTest.py

```diff
@@ -856,3 +863,24 @@ def create_arguments(filename, file, config_file, some_value_B):
         self.assertEqual(create_arguments_mock.call_args[0][2][-5:], '.conf')
         generate_config_mock.assert_called_once_with(
             self.testfile2_path, self.testfile2_content, 124)
+
+    def test_capture_groups_warnings(self):
+        logger = logging.getLogger()
+        with self.assertLogs(logger, 'WARNING') as cm:
+            @linter('some-executable',
+                    use_stdout=True,
+                    output_format='regex',
+                    output_regex=r'(?P<not_supported_name>)\d+(\w+)')
+            class SomeBear:
+                pass
```

Comment:
Can't use `EmptyLinterBear` in here, it needs to be defined right below the decorator it seems